dotnet/src/SemanticKernel.Abstractions/SemanticKernel.Abstractions.csproj

```diff
@@ -22,6 +22,9 @@
     <PackageReference Include="Microsoft.Bcl.HashCode" />
```

Comment:
Maybe these references could also be `netstandard2.0` only

Reply:
Thanks for the tip, I have gone and made the necessary optimisations.